WatchParty/backend/models/user.js

```diff
@@ -0,0 +1,29 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('User', {
+    email: {
+      type: DataTypes.STRING(32),
+      allowNull: false,
+      unique: true,
+      validate: { isEmail: true }
+    },
+    firstName: {
+      type: DataTypes.STRING(32),
+      allowNull: false,
+      validate: { notNull: true }
+    },
+    lastName: {
+      type: DataTypes.STRING(32),
+      allowNull: false,
+      validate: { notNull: true }
+    },
+    password: {
+      type: DataTypes.STRING(),
+      allowNull: false,
+      validate: { notNull: true, len: [8, 256] }
+    }
+  }, {})
+  User.associate = function(models) {
+    // associations can be defined here
+  }
+  return User
+}
```

Comment:
The user.js model shares parameters with the users.js model in Thomas's PR, and its name is also very similar (one letter difference). We should make the names of these two models more unique to prevent future confusion.

Reply:
Thanks @amckoy42 . I'll pay attention to that difference when we incorporate @usersblock 's work. According the sequelize docs, model names are supposed to be singular and the table name (which is taken care of for us) is plural. https://sequelize.org/v5/manual/getting-started.html#modeling-a-table